### ISSUE TITLE
Tab order 1 

### ISSUE THREAD

**Issue by A11YAngel** [opena11y/fae-ainspector#12]
Forgot to add this other part to the previous Tab order issue: 

However, the focus should be on the Selected Rule window first (for rendering information) then the Details button should be last. See the visual reading order with two options. There are two options for the Details button: keep the button in the same location as a third item in the tab order or move the button to the bottom as a third item. 

![details button 2](https://cloud.githubusercontent.com/assets/10927103/6196014/b329a9b4-b399-11e4-8946-f9c03e678d16.jpg)
